src/app/maptools/qgsappmaptools.h

```diff
@@ -113,6 +113,7 @@ class QgsAppMapTools
       ChangeLabelProperties,
       ReverseLine,
       TrimExtendFeature,
+      EditMesh
```

Comment:
Can I suggest everywhere EditMesh->EditMeshFrame (everywhere class name , etc?). since I assume that for editing mesh result we would have different tool anyways